CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # bedrock-vue-wallet ChangeLog
 
+## 19.0.0 - 2023-TBD
+
+### Changed
+- **BREAKING**: Update peer dep `@bedrock/web-wallet` to v11.1.
```

Comment:
This isn't a breaking change because bedrock-web-wallet's update was minor. I don't think we actually need a PR for this at this level -- we can have top-level apps require v11.1 though.

Reply:
Yep, this is not required. I'll close it. Thanks